katas/content/multi_qubit_systems/bell_state_change_1/solution.md

```diff
@@ -0,0 +1,21 @@
+We recognize that the goal is another Bell state. In fact, it is one of the four Bell states.
+
+We remember from "Sign flip" that the Pauli Z gate will change the state of the $|0\rangle$ basis state of a single qubit, so this gate seems like a good candidate for what we want to achieve. This gate leaves the sign of the $|0\rangle$ basis state of a superposition unchanged, but flips the sign of the $|1\rangle$ basis state of the superposition.
+
+Don't forget that the Z gate acts on only a single qubit, and we have two here.
+Lets also remember how the Bell state is made up from its individual qubits.
+
+If the two qubits are A and B, where A is `qs[0]` and B is `qs[1]`, we can write that
+$|\Phi^{+}\rangle = \frac{1}{\sqrt{2}} \big(|0_{A}0_{B}\rangle + |1_{A}1_{B}\rangle\big)$, and also remember that this is a superposition.
```

Comment:
I'm not sure what is the point of reminding that this is a superposition without an explanation, so let's remove this
```suggestion
$|\Phi^{+}\rangle = \frac{1}{\sqrt{2}} \big(|0_{A}0_{B}\rangle + |1_{A}1_{B}\rangle\big)$.
```